### ISSUE TITLE
Add support for specifying the commit ref in the config file

### ISSUE THREAD

**Issue by andrewrynhard** [talos-systems/conform#188]

**Comment by olivernybroe** [comment]
This feature is especially handy when it comes to GitHub actions and also in general for sharing the config file with the rest of the team

**Comment by andrevalentin** [comment]
Yeah this would be perfect, really appreciating all your support. Adding this would save us from using another action for this particular check presently. 💯
